docs/developing.rst

```diff
@@ -299,8 +299,8 @@ Release a new version of edx-proctoring
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * Update the version in ``edx_proctoring/__init__.py`` and ``package.json``
+* Describe your changes in `CHANGELOG.rst`
 * Create a `new release on GitHub <https://github.com/edx/edx-proctoring/releases>`_ using the version number
-* Send an email to release-notifications@edx.org announcing the new version
 * Update edx-platform to use the new version
     * In edx-platform, create a branch and update the requirements/edx/base.in file to reflect the new tagged branch.
```

Comment:
I think this should also reference updating development.txt and testing.txt as well.